app/models/vegetable.rb

```diff
@@ -10,6 +10,6 @@ def as_json(options = {})
     end
 
     average_rating = total/number_of_ratings
-    super(options).merge(average_rating: average_rating)
```

Comment:
average_rating should be added to 'options' before calling the parent class ApplicationRecord. My change would fix this.